test/converters/xcresult3/converter_test.go

```diff
@@ -47,66 +47,60 @@ func TestConverter_XML(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, []junit.TestSuite{
 			{
-				Name: "BullsEyeTests", Tests: 5, Failures: 0, Skipped: 0, Errors: 0, Time: 0.9777920246124268,
+				Name: "BullsEyeTests", Tests: 5, Failures: 0, Skipped: 0, Errors: 0, Time: 0.9774,
 				TestCases: []junit.TestCase{
 					{
 						Name: "testStartNewRoundUsesRandomValueFromApiRequest()", ClassName: "BullsEyeFakeTests",
-						Time: 0.014459013938903809,
+						Time: 0.014,
 					},
 					{
 						Name: "testGameStyleCanBeChanged()", ClassName: "BullsEyeMockTests",
-						Time: 0.00929105281829834,
+						Time: 0.0093,
 					},
 					{
 						Name: "testScoreIsComputedPerformance()", ClassName: "BullsEyeTests",
-						Time: 0.7373920679092407,
+						Time: 0.74,
 					},
 					{
 						Name: "testScoreIsComputedWhenGuessIsHigherThanTarget()", ClassName: "BullsEyeTests",
-						Time: 0.004113912582397461,
+						Time: 0.0041,
 					},
 					{
 						Name: "testScoreIsComputedWhenGuessIsLowerThanTarget()", ClassName: "BullsEyeTests",
-						Time: 0.21253597736358643,
+						Time: 0.21,
 					},
 				},
 			},
 			{
-				Name: "BullsEyeSlowTests", Tests: 2, Failures: 0, Skipped: 0, Errors: 0, Time: 0.5334550142288208,
+				Name: "BullsEyeSlowTests", Tests: 2, Failures: 0, Skipped: 0, Errors: 0, Time: 0.53,
 				TestCases: []junit.TestCase{
 					{
 						Name: "testApiCallCompletes()", ClassName: "BullsEyeSlowTests",
-						Time: 0.2844870090484619,
+						Time: 0.28,
 					},
 					{
 						Name: "testValidApiCallGetsHTTPStatusCode200()", ClassName: "BullsEyeSlowTests",
-						Time: 0.2489680051803589,
+						Time: 0.25,
 					},
 				},
 			},
 			{
-				Name: "BullsEyeUITests", Tests: 1, Failures: 0, Skipped: 0, Errors: 0, Time: 9.606701016426086,
+				Name: "BullsEyeUITests", Tests: 1, Failures: 0, Skipped: 0, Errors: 0, Time: 9,
 				TestCases: []junit.TestCase{
 					{
 						Name: "testGameStyleSwitch()", ClassName: "BullsEyeUITests",
-						Time: 9.606701016426086,
+						Time: 9,
 					},
 				},
 			},
 			{ // testFlakyFeature() was flaky: failed once, and then passed the second run
-				Name: "BullsEyeFlakyTests", Tests: 3, Failures: 1, Skipped: 1, Errors: 0, Time: 0.22202682495117188,
+				Name: "BullsEyeFlakyTests", Tests: 2, Failures: 0, Skipped: 1, Errors: 0, Time: 0.12,
```

Comment:
`Tests` have changed from 3 to 2, is this intentional? Also, the comment doesn't seem in sync with the test case expectation.

Reply:
Yes, because the new extraction method handles the retried tests better and only shows the final state for them.